### PR TITLE
Tweet on new additions (and the older ones too once in a while)

### PR DESCRIPTION
**Pull request by RJ722**
Closes: #4﻿

### CODE REVIEW

File: gfi/tweet.py [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+            name not in self.tweeted_already or not self.tweeted_recently(name)
+        )
+
+    def tweet(self, repo):
+        try:
+            status = (
+                f"{ repo['name'] } needs your help. Start contributing to "
+                f" issues marked #goodfirstissues here. {repo['url']} "
+                f" #oss #{repo['language']}"
+            )
```
Comment (RJ722) [inline gfi/tweet.py:80]: @sanketsaurav suggest a better tweet?

**Comment by RJ722** [comment]
@jaipradeesh @sanketsaurav gentle nudge. Can I get a review on this?

**Comment by kaustavdm** [comment]
Updates on this?

**Comment by RJ722** [comment]
DeepSource and other CI tests are green. As soon as I get twitter credentials from @jaipradeesh, I'll add relevant keys to Vercel, and we can go ahead and merge this.

**Comment by jaipradeesh** [comment]
Update: Still waiting to hear back from Twitter on the status of the developer a/c application.

**Comment by jaipradeesh** [comment]
Followed up with Twitter. Not sure why the delay.

**Comment by bdougie** [comment]
FWIW it looks like Twitter most likely approved your application but never sent an email.

I received the following email today for my application

> We identified an issue in our system where people who recently applied for a developer account may not have received a confirmation email informing them of their application approval. Your application was among those affected. This issue has been resolved.

**Comment by jaipradeesh** [comment]
Appreciate it, @bdougie The application was indeed approved but we weren't notified. This PR should go live today or tomorrow.

**Pull request merged**